source/node/tools/command.js

```diff
@@ -1,39 +1,96 @@
+    DeleteEntry: { s: "den", d: true, args: [ARG.ItemID] },
+    DeleteEntryAttribute: { s: "dea", d: true, args: [ARG.ItemID, ARG.StringValue] },
+    DeleteEntryMeta: { s: "dem", d: true, args: [ARG.ItemID, ARG.StringValue] },
+    DeleteEntryProperty: { s: "dep", d: true, args: [ARG.ItemID, ARG.StringValue] },
+    DeleteGroup: { s: "dgr", d: true, args: [ARG.ItemID] },
+    DeleteGroupAttribute: { s: "dga", d: true, args: [ARG.ItemID, ARG.StringValue] },
+    Format: { s: "fmt", d: false, args: [ARG.StringValue] },
+    MoveEntry: { s: "men", d: false, args: [ARG.ItemID, ARG.ItemID] },
+    MoveGroup: { s: "mgr", d: false, args: [ARG.ItemID, ARG.ItemIDOrRoot] },
+    Pad: { s: "pad", d: false, args: [ARG.ItemID] },
+    SetArchiveAttribute: { s: "saa", d: false, args: [ARG.StringValue, ARG.StringValue] },
+    SetEntryAttribute: { s: "sea", d: false, args: [ARG.ItemID, ARG.StringValue, ARG.StringValue] },
+    SetEntryMeta: { s: "sem", d: false, args: [ARG.ItemID, ARG.StringValue, ARG.StringValue] },
+    SetEntryProperty: { s: "sep", d: false, args: [ARG.ItemID, ARG.StringKey, ARG.StringValue] },
+    SetGroupAttribute: { s: "sga", d: false, args: [ARG.ItemID, ARG.StringValue, ARG.StringValue] },
+    SetGroupTitle: { s: "tgr", d: false, args: [ARG.ItemID, ARG.StringValue] }
+};
+
 /**
- * Command related tools
- * @module command
+ * Extract command components from a string
+ * @param {String} command The command to extract from
+ * @returns {String[]} The separated parts
  */
-module.exports = {
-    /**
-     * Extract command components from a string
-     * @param {String} command The command to extract from
-     * @returns {String[]} The separated parts
-     */
-    extractCommandComponents: function(command) {
-        var patt = /("[^"]*")/,
-            quotedStringPlaceholder = "__QUOTEDSTR__",
-            escapedQuotePlaceholder = "__ESCAPED_QUOTE__",
-            matches = [],
-            match;
+function extractCommandComponents(command) {
+    var patt = /("[^"]*")/,
+        quotedStringPlaceholder = "__QUOTEDSTR__",
+        escapedQuotePlaceholder = "__ESCAPED_QUOTE__",
+        matches = [],
+        match;
+
+    command = command.replace(/\\\"/g, escapedQuotePlaceholder);
 
-        command = command.replace(/\\\"/g, escapedQuotePlaceholder);
+    while ((match = patt.exec(command))) {
+        var matched = match[0];
+        command =
+            command.substr(0, match.index) + quotedStringPlaceholder + command.substr(match.index + matched.length);
+        matches.push(matched.substring(1, matched.length - 1));
+    }
 
-        while ((match = patt.exec(command))) {
-            var matched = match[0];
-            command =
-                command.substr(0, match.index) + quotedStringPlaceholder + command.substr(match.index + matched.length);
-            matches.push(matched.substring(1, matched.length - 1));
+    var parts = command.split(" ");
```

Comment:
and maybe rest of them? I understand that this is just reformatting but maybe while we're at it :D